modules/console/pages/config/security/authentication.adoc

```diff
@@ -1,61 +1,357 @@
+
+Then, <<api-authentication, configure API authentication>> for specific APIs.
+
+=== Enable basic authentication
+
+To enable basic authentication, configure your authentication settings in the Redpanda Console configuration file.
+
+[, yaml]
+----
+authentication:
+  jwtSigningKey: <signing key> <1>
+  useSecureCookies: false <2>
+  # Basic auth logins are mapped to SASL-SCRAM for Kafka.
+  basic:
+    enabled: true <3>
+----
+<1> `authentication.jwtSigningSecret` (required): The secret key for signing JWTs. Keep it confidential. Minimum 32 characters. Use xref:console:config/configure-console.adoc#environment-variables[environment variables] or command-line flags for sensitive data.
+<2> `authentication.useSecureCookies`: A boolean flag indicating whether the JWT cookie should be marked as secure. Set to true in production to enforce HTTPS.
+<3> `authentication.basic.enabled` (required): Enables or disables basic (username/password) authentication. When enabled, credentials are mapped to SASL-SCRAM for Kafka.
+
+Then, <<api-authentication, configure API authentication>> for specific APIs.
+
+[[api-authentication]]
+== Configure API authentication
+
+After you <<enable-authentication, enable authentication>> for Redpanda Console, you must configure how Redpanda Console authenticates to the following APIs in your Redpanda cluster:
+
+* **Kafka API:** Enables core messaging operations and authenticates requests to your Kafka cluster.
+* **Schema Registry API:** Allows Redpanda Console to manage and display schema information.
+* **Admin API:** Unlocks management features such as viewing cluster details, managing users, and generating debug bundles.
+
+Choose one of the following authentication methods for each API:
+
+* **User impersonation:**
+  Redpanda Console uses the same credentials you use to log into Redpanda Console to authenticate API requests to Redpanda. This authentication method ensures accurate audit trails in xref:manage:audit-logging.adoc[Redpanda audit logs] and unified authorization across both Redpanda and Redpanda Console. Your credentials authenticate with any <<api-authentication, API where `impersonateUser` is `true`>>.
+  +
+  In this mode, Redpanda Console's role-based access control (RBAC) is bypassed for the impersonated API calls.
+
+* **Static credentials:**
+  Use preconfigured credentials to authenticate to Redpanda and use role bindings for access control in Redpanda Console. This authentication method is useful when impersonation is disabled or when you need to separate Redpanda Console RBAC from the logged-in user's identity in Redpanda.
+
+=== Kafka API
+
+When user impersonation is enabled, the logged-in user's credentials are used for authentication.
+
+.User impersonation
```

Comment:
Along with Static credentials below, this introduces a different (others throughout the doc typically look like "User impersonation example:") way to introduce the code examples.